docs/questions/native-editor/filter-widgets.md

```diff
@@ -0,0 +1,110 @@
+---
+title: Filter and parameter widgets for native code
+---
+
+# Filter and parameter widgets for native code
+
+When you add a [SQL variable or parameter](./sql-parameters.md) to your native/SQL query, Metabase will add a widget to the top of the query that people can use to set the variable's value.
+
+## How to create different types of filter and parameter widgets
```

Comment:
In general, it's not that simple. There are interactions between metadata and filter type, and there's also dependency on how many records a field has.
E.g. if a field is set to "search box" in table metadata, it will not show up as a dropdown even if you select that on a question level.